spec/plugin.spec.js

```diff
@@ -615,7 +620,7 @@ describe('ManifestPlugin', function() {
         }
       }, function(manifest){
         expect(manifest).toEqual({
-          'javascripts/main.js': 'javascripts/main.js'
+          [`javascripts${path.sep}main.js`]: 'javascripts/main.js'
```

Comment:
note to self: Is this the expected behaviours?

Reply:
yep, it should stay 'javascripts/main.js', are you having issues on windows?

It should be covered by: https://github.com/danethurber/webpack-manifest-plugin/blob/421fa29716cdb26251c10e90c6d32ecc56b6a6c8/lib/plugin.js#L152-L156

Reply:
The output in Windows is
```
{
    'javascripts\main.js': 'javascripts/main.js'
}
```

I found it odd that the value is changed byt not the key. If this isn't the expected behavoir, then I guess it's an unrelated bug. I'll revert this test change.

Reply:
I tried overwriting `path.sep` to simulate a windows environment, and the test pass

Could you double check it on your side?

Reply:
Looks like the `opts.map` provided by the UT itself was changing the file name and using the OS path sep. I updated `plugin.js` to standardize paths again after opts.map is run as I figure you want the paths standardized both before `opts.filter` and after `opts.map`